ENG-622 - 1.60.0 - OIDC prompt parameter (#3891)
FusionAuth/fusionauth-site: astro/src/content/docs/lifecycle/authenticate-users/oauth/tokens.mdx

    <AuthenticationTypeValues />
  </APIField>
  <APIField name="auth_time" type="Long" since="1.36.0">

>     The time of the initial authentication request expressed as [unix time](/docs/reference/data-types#unix-time). This claim will remain the same even when the token has been re-issued through the use of a Refresh Token.

Reviewer: ```suggestion
    The time of the initial authentication request expressed as [unix time](/docs/reference/data-types#unix-time). This claim remains the same even if the token is re-issued using a Refresh Token.
```

    In version `1.50.0` and later, when the `Scope handling policy` is `Strict`, this field is only populated when the <InlineField>scope</InlineField> claim contains the `email` scope.
  </APIField>
  <APIField name="email_verified" type="Boolean">

>     The OpenID Connect claim indicating if the User's email has been verified.

Reviewer: ```suggestion
    An OpenID Connect claim that's `true` then the User's email address is verified.
```

    This field is only populated when the `Scope handling policy` is `Strict`, and the <InlineField>scope</InlineField> claim contains the `phone` scope.
  </APIField>
  <APIField name="phone_number_verified" type="Boolean" since="1.59.0">

>     The OpenID Connect claim indicating if the User's phone number has been verified. This claim is based only on the <InlineField>user.phoneNumber</InlineField> verification status.

Reviewer: ```suggestion
    An OpenID Connect claim that's `true` if the User's phone number is verified. This claim is based only on the <InlineField>user.phoneNumber</InlineField> verification status.
```
